### PR TITLE
MM-13091 Review, improve and test the push proxy docker image

### PR DESCRIPTION
**Pull request by stylianosrigas**
> MM-13091 Review, improve and test the push proxy docker image
 
- Current Dockerfile follows best practises
- Ιmpovements made include:
  - Change of PP_VERSION variable to APP_VERSION 
  - Combining of all Docker Volume creation entries in a single entry, 
  - Move of ENTRYPOINT entry to the end of the Dockerfile to follow Dockerfile best practises.
- The docker image was recreated and tested to make sure the server works as expected.

Fixes https://github.com/mattermost/mattermost-server/issues/10314

### CODE REVIEW

File: docker/push-proxy/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:16.04
```
Comment (cpanato) [inline docker/push-proxy/Dockerfile]: can we use another image like `alpine`?

@jwilander do you know if exist any external dependencies?

Comment (jwilander) [reply docker/push-proxy/Dockerfile]: None that I'm aware of

Comment (stylianosrigas) [reply docker/push-proxy/Dockerfile]: @cpanato @jwilander I will test with alpine and update ;)

Comment (stylianosrigas) [reply docker/push-proxy/Dockerfile]: I have tested with Alpine and it was causing issues to the existing push-proxy application. Therefore, I moved to using version 18.04 of Ubuntu, which is the new Ubuntu Minimal version and more details can be found here -> https://blog.ubuntu.com/2018/07/09/minimal-ubuntu-released. The total size of this image is 29MB.

**Pull request merged**